build/cleanup-images.yml

```diff
@@ -0,0 +1,55 @@
+# DESCRIPTION: 	
+# Removes old docker images.
+
+trigger: none
+pr: none
+
+variables:
+- template: build-variables.yml
+
+jobs:
+- job: deleteImage
+  displayName: 'Delete Images'
+  pool:
+    vmImage: $(WindowsVmImage)
+  steps:
+  - task: AzureCLI@2
+    displayName: 'Delete Images'
+    imputs:
+      azureSubscription: $(ConnectedServiceName)
+      scriptType: ps
+      scriptLocation: InlineScript
+      inlineScript: |
+        $versions = "stu3","r4","r5"
+        $tagList = az acr repository show-tags --name $(azureContainerRegistryName) --repository r4_fhir-server
```

Comment:
Is this supposed to be `r4_fhir-server`?

Reply:
Yes. Since each repo has the same images sent to them I was using one of them as the source of truth for the tags to avoid redundant requests.

Reply:
Changed it to look up the list of tags for each version separately.